angular-workspace/projects/ni/nimble-angular/src/directives/breadcrumb-item/nimble-breadcrumb-item-router-link-with-href.directive.ts

```diff
@@ -20,4 +22,28 @@ export class NimbleBreadcrumbItemRouterLinkWithHrefDirective extends RouterLinkW
     public set nimbleRouterLink(commands: never[] | string | null | undefined) {
         this.routerLink = commands;
     }
+
+    public constructor(injector: Injector, private readonly elementRef: ElementRef<BreadcrumbItem>) {
+        super(injector.get(Router), injector.get(ActivatedRoute), injector.get(LocationStrategy));
+    }
+
+    public override onClick(_button: number, _ctrlKey: boolean, _shiftKey: boolean, _altKey: boolean, _metaKey: boolean): boolean {
+        // onClick is the 'click' @HostListener handler in RouterLinkWithHref. Override it to do nothing so that this directive
+        // can be in control of when the click handler is called. This allows RouterLinkWithHref's onClick to only be called
+        // when the link within the element is clicked rather than any part of the element.
+        return true;
+    }
+
+    @HostListener(
+        'click',
+        ['$event', '$event.button', '$event.ctrlKey', '$event.shiftKey', '$event.altKey', '$event.metaKey']
+    )
+    public breadcrumbItemClick(event: MouseEvent): boolean {
```

Comment:
Is this notably different than just having this logic in the `onClick` override above and just returning false in the case we don't want to handle it?

Reply:
We need access to the actual `$event` so that we can tell whether or not the click occurred on the anchor element. `onClick` doesn't provide us access to that.